File: src/main/kotlin/application/controller/dialogs/rule/isotropic/INTEnergeticDialog.java
package application.controller.dialogs.rule.isotropic;

import application.controller.dialogs.rule.RuleWidget;
import application.model.rules.isotropic.rules.energetic.INTEnergetic;

public class INTEnergeticDialog extends RuleWidget {
    public INTEnergeticDialog() {
        super();

        ruleFamily = new INTEnergetic();
    }

    @Override
    public void updateRule(String rulestring) {
        ruleFamily.setRulestring(rulestring);
    }
}

File: src/main/kotlin/application/controller/dialogs/rule/isotropic/INTEnergeticDialog.kt
package application.controller.dialogs.rule.isotropic

import application.controller.dialogs.rule.RuleWidget
import application.model.rules.isotropic.rules.energetic.INTEnergetic

class INTEnergeticDialog : RuleWidget() {
    override fun updateRule(rulestring: String?) {
        ruleFamily!!.rulestring = rulestring
    }

    init {
        ruleFamily = INTEnergetic()
    }
}